src/panels/calendar/ha-full-calendar.ts

```diff
@@ -292,6 +292,7 @@ export class HAFullCalendar extends LitElement {
   static get styles(): CSSResult[] {
     return [
       haStyle,
+      haStyleScrollbar,
```

Comment:
Do we still need this when we add the styles manually and removed the class?

Reply:
I think you reviewed old code